tensorflow_model_optimization/g3doc/guide/clustering/clustering_comprehensive_guide.ipynb

```diff
@@ -0,0 +1,536 @@
+{
```

Comment:
<p>I suggested adding this, but from discussion with others</p><p> </p><ol><li>I realize it would require modifying <code>strip_clustering/pruning</code>  to also support striping the wrapper from a layer, which is a API change that requires more discussion.</li><li>We need to do more thorough testing to ensure overriding the attribute works in all meaningful cases (I've started doing this).</li></ol><p> </p><p>In the first release of the docs, we can omit this, but keep it in mind.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/model-optimization/pull/330/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>OK, this is deleted. Thanks.</p>